sigh/lib/sigh/runner.rb

```diff
@@ -65,7 +65,7 @@ def profile_type
     # Fetches a profile matching the user's search requirements
     def fetch_profiles
       UI.message "Fetching profiles..."
-      results = profile_type.find_by_bundle_id(Sigh.config[:app_identifier]).find_all(&:valid?)
+      results = profile_type.find_by_bundle_id(Sigh.config[:app_identifier])
```

Comment:
But it still has to be valid for us to download. Even if we repair it, something might go wrong on either fastlane's or Apple's side, so I'd say we should leave this here. It's better to fail early, than later at the build process